src/cdomains/lval.ml

```diff
@@ -355,11 +346,13 @@ struct
 
   let printXml f x = BatPrintf.fprintf f "<value>\n<data>\n%a\n</data>\n</value>\n" printInnerXml x
 
+  (* TODO: only used for unused musteqdomain *)
```

Comment:
The `musteqdomain` has some sort of TODO that it should e.g. be used for `regions`

Reply:
Yup, I moved these few functions from here to there, so all that unused stuff is together.